lib/circuit/CircuitOptions.ts

```diff
@@ -41,17 +41,15 @@ export type CircuitOptions = {
   errorFilter?: (err: Error) => boolean,
 };
 
-/**
- * A circuit breaker.
- * After the method fails `threshold` count it enters the closed state and
- * throws a `Circuit closed.` error. Once in closed state, the circuit fails
- * for the provided `timeout` milliseconds. After the `timeout` interval expires
- * the circuit transitions to half-opened state and allows next execution.
- * If the execution succeeds then circuit transitions back to open state and resets
- * the number of counted errors to zero.
- * @param threshold the max number of failures until the circuit gets closed.
- * @param timeout timeout in milliseconds to keep the circuit in closed state.
- */
-export function circuit(threshold: number, timeout: number, options?: CircuitOptions) {
-  throw new Error('Not implemented.');
-}
+export const DEFAULT_INTERVAL = undefined;
+export const DEFAULT_POLICY = 'errors';
+export const DEFAULT_ON_ERROR = 'throw';
+export const DEFAULT_SCOPE = 'class';
+export const DEFAULT_ERROR_FILTER = () => true;
```

Comment:
Maybe we can get rid of these, and just provide defaults directly in the `DEFAULT_OPTIONS`